solana/solana-ibc/programs/solana-ibc/src/tests.rs

```diff
@@ -87,6 +88,8 @@ fn anchor_test_deliver() -> Result<()> {
     let chain =
         Pubkey::find_program_address(&[crate::CHAIN_SEED], &crate::ID).0;
 
+    let hashed_denom = hash::hash(BASE_DENOM.as_bytes()).to_bytes().to_vec();
```

Comment:
As before, why not CryptoHash?
```suggestion
    let hashed_denom = CryptoHash::digest(BASE_DENOM.as_bytes());
```